ansible/roles/eap7-dev/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+######################### Install Software Packages
+
+- name: Install Software Packages
+  import_tasks: ./packages.yml
+  tags:
+    - install_software_packages
+
+- file:
+    path: "/home/{{student_user}}"
+    state: directory
+    owner: "{{student_user}}"
+    mode: 0755
+
+- file:
+    path: "/srv"
+    state: directory
+    owner: "{{student_user}}"
+    mode: 0755
+
+- name: download apache-maven-3.6.3
+  ignore_errors: true
+  get_url:
+    url: ftp://apache.cs.utah.edu/apache.org/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz
```

Comment:
Please use https://gpte-public.s3.amazonaws.com/apache-maven-3.6.3-bin.tar.gz instead. We don't want to rely on the Apache FTP server...